biz.aQute.resolve/src/biz/aQute/resolve/DistroRepository.java

```diff
@@ -0,0 +1,21 @@
+package biz.aQute.resolve;
+
+import java.io.File;
+
+import org.osgi.resource.Resource;
+
+import aQute.bnd.osgi.Domain;
+import aQute.bnd.osgi.repository.ResourcesRepository;
+import aQute.bnd.osgi.resource.ResourceBuilder;
+
+public class DistroRepository extends ResourcesRepository {
```

Comment:
-runsystemcapabilities can't be considered when in `distro` mode.  See https://github.com/bndtools/bnd/issues/1894